.gitignore

```diff
@@ -69,4 +69,8 @@ Config.toml
 *.idea
 
 # .ballerina files
-*.ballerina
\ No newline at end of file
+*.ballerina
+
+# Dependency File
+ballerina/
+*Dependencies.toml
```

Comment:
Let's not add the `dependencies.toml` to gitignore, as we are yet to decide whether we should commit it or not.

Reply:
Removed now.